docs/api/api_blueprint.apib

```diff
@@ -24175,6 +24175,22 @@ Get a list of all orders
           }
         }
 
+## Get Order Invoice for an event [/v1/orders/invoices/{order_identifier}]
++ Parameters
+    + order_identifier: example (string) - Identifier of the order
```

Comment:
I'm doubtful about the value I need to place instead of `example` in the docs. @uds5501 @shreyanshdwivedi @iamareebjamal